pkg/runtime/runtime.go

```diff
@@ -231,13 +234,23 @@ func DefaultInitRuntimeStage(o *runtimeOptions, m *MosnRuntime) error {
 		return err
 	}
 	// init all kinds of components with config
-	if err := m.initCustomComponents(o.services.custom); err != nil {
+	//init secret & config first
+	if err := m.initSecretStores(o.services.secretStores...); err != nil {
```

Comment:
Now the secret stores are inited twice:
![image](https://user-images.githubusercontent.com/26001097/177329220-89bc9333-4315-426a-9fcf-01c2023640df.png)
We should remove line 280